server.js

```diff
@@ -0,0 +1,56 @@
+const express = require("express");
+const app = express();
+const port = 9090;
+const fs = require("fs").promises;
+
+app.use(express.json());
+
+//----bringing data-------
+
+// ---------   Createing routes  ------------
+
+// reading All albums
+app.get("/albums", (req, res) => {
+  fs.readFile("./albums.json", "utf-8").then((movie) => {
```

Comment:
You don't write to this file, so it doesn't neccessarily make sense to read it everytime you make a request. require() will do the job